animation_nodes/nodes/gpencil/gp_material_output.py

```diff
@@ -0,0 +1,48 @@
+import bpy
```

Comment:
I think this node should use the `isUsed` socket mechanism.

Reply:
Okay. I'll make the changes.